rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1711,6 +1871,147 @@ def assert_binance_import_results(rotki: Rotkehlchen):
+            notes='Deposit in Staking Purchase',
+        ), HistoryEvent(
+            identifier=21,
+            event_identifier='BNC_4259a864eeb6e39a046bbd4030a62d63bcaafebb36c4a5f55b00d1e78a90569e',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1686543089)),
+            location=Location.BINANCE, event_type=HistoryEventType.STAKING,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_ETH_MATIC,
+            balance=Balance(amount=FVal(602.5193197)),
+            location_label='CSV import',
+            notes='Deposit in Simple Earn Locked Subscription',
+        ), HistoryEvent(
+            identifier=22,
+            event_identifier='BNC_4b706868b7c8906507b66b1b35017dcaa9eec3c27e63563603e8aa7475243f81',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1686824951)),
+            location=Location.BINANCE, event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=A_BUSD,
+            balance=Balance(amount=FVal(0.35628294)),
+            location_label='CSV import',
+            notes='Reward from Cash Voucher Distribution',
+        ), HistoryEvent(
+            identifier=23,
+            event_identifier='BNC_6b282ed5ab3de42a924f008ed20e35ebcbbe1618b60c0a120c7c76278240f725',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1686825000)),
+            location=Location.BINANCE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=A_BUSD,
+            balance=Balance(amount=FVal(4.87068)),
+            location_label='CSV import',
+            notes='Reward from Mission Reward Distribution',
+        ), HistoryEvent(
+            identifier=24,
+            event_identifier='BNC_d6a38298147f2c10d888205893ac2baeccad83d1dbed1477a3e2f4ddd943761f',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1694505602)),
+            location=Location.BINANCE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(0.07779065)),
+            location_label='CSV import',
+            notes='0.07779065 USDT fee paid on binance USD-MFutures',
+        ), HistoryEvent(
+            identifier=28,
+            event_identifier='BNC_33b05944ecfd96a56834fce7787e8bbc264b43846f56b50c633d6ceb525044e2',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1694623421)),
+            location=Location.BINANCE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.PAYBACK_DEBT,
+            asset=A_USDT,
+            balance=Balance(amount=FVal('1.29642000')),
+            location_label='CSV import',
+            notes='1.29642000 USDT realized loss on binance USD-MFutures',
+        ), HistoryEvent(
```

Comment:
Why did you add this even in the CSV for the test? It's same type as above. So already tested.